src/app/activitywidget.ui

```diff
@@ -21,33 +21,90 @@
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbCustomer">
+       <property name="sizePolicy">
```

Comment:
I will check on linux how it impacts rendering. It is ok under windows.

Reply:
I'm using it on Linux. It looks good.